Makefile

```diff
@@ -7,10 +7,10 @@ client-libs-docs:
 	@for d in $(shell ls -d content/docs/*); do \
 		/bin/bash -c "pushd $$d/; \
 			rm -f client-libraries.md client-features.md; \
-			ln -s ../../_client_libs/client-libraries.md; \
-			ln -s ../../_client_libs/client-features.md; \
+			cp ../../_client_libs/client-libraries.md ./; \
+			cp ../../_client_libs/client-features.md ./; \
 			popd"; \
-		echo "sym-linked content/_client_libs/*.md -> $$d/"; \
+		echo "copied content/_client_libs/*.md -> $$d/"; \
```

Comment:
why did you need to chance symlinking to copying?

Reply:
Because in Hugo 0.123.x symlink won't work for the input files; in other words, it won't generate files.

FYI: https://github.com/gohugoio/hugo/issues/11556

Reply:
If this uses symlinks, this page will be 404.

`https://deploy-preview-675--romantic-neumann-1959d7.netlify.app/docs/<version>/client-features/`

https://deploy-preview-675--romantic-neumann-1959d7.netlify.app/docs/1.54/client-libraries/
https://deploy-preview-675--romantic-neumann-1959d7.netlify.app/docs/1.54/client-features/

Reply:
ok. But in this case you should simplify the loop, you don't need rm/pushd/popd, just the copy commands

Reply:
Done!